src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponentSpec.scala

```diff
@@ -0,0 +1,122 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import java.util.UUID
+
+import org.broadinstitute.dsde.rawls.dataaccess.SlickWorkspaceContext
+import org.broadinstitute.dsde.rawls.model._
+import org.joda.time.DateTime
+
+/**
+ * Created by mbemis on 2/22/16.
+ */
+class SubmissionComponentSpec extends TestDriverComponent {
```

Comment:
these are the ported test cases. I feel like there should be a few more that deal with workflow failures. I'll add them before this PR goes in